src/log/log.c

```diff
@@ -2086,7 +2078,7 @@ __wt_log_scan(WT_SESSION_IMPL *session, WT_LSN *start_lsnp, WT_LSN *end_lsnp, ui
      */
     lastlog = 0;
     if (log != NULL) {
-        allocsize = log->allocsize;
```

Comment:
The `allocsize` assignment can be lifted out of the `if`, since both branches now do the same thing.